model-optimizer/mo/front/extractor.py

```diff
@@ -132,13 +132,13 @@ def attr_getter(node: Node, name: str):
 
 
 def bool_to_str(node: Node, attr: str):
-    # Function converts 0/1 or bool False/True values to str 'false'/'true' which need to appear in IR
+    # Function converts 0/1 or bool False/True or '0'/'1' values to str 'false'/'true' which need to appear in IR
     attribute_name = node.soft_get(attr, None)
     if attribute_name is None:
         return None
     if isinstance(attribute_name, bool):
         return str(attribute_name).lower()
-    elif attribute_name in [0, 1]:
+    elif attribute_name in [0, 1, '0', '1']:
         return str(bool(attribute_name)).lower()
```

Comment:
here you try to cast string (or char) to bool. I think this is not correct. I just checked '0'->True and '1'-> True

Reply:
good catch, last minute change rare gives good result (((